src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+.main__container {
+  display: grid;
+  gap: 46px 48px;
+  padding: 50px 40px;
+  place-content: center;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  border: 1px solid $color-gray;
+  width: 200px;
+  padding: 32px 16px 16px;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    width: 100%;
+    color: $text-main-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-top: 0;
+    margin-bottom: 4px;
+    transition: color 300ms ease;
+
+    &:hover {
+      color: #34568b;
+    }
+  }
+  &__code {
+    color: $text-secondary-color;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__reviews {
+    color: $text-main-color;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__stars {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+    margin-top: 16px;
+  }
+
+  &__cost {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+  &__price {
+    color: $text-secondary-color;
+    font-size: 12px;
+    line-height: 18px;
+  }
+  &__total {
+    color: $text-main-color;
+    line-height: 18px;
+    font-weight: 700;
+    text-align: right;
+  }
+
+  &__button {
+    width: 100%;
+    justify-content: center;
+    text-align: center;
+    text-transform: uppercase;
+    font-weight: 700;
+    height: 40px;
+    border-radius: 5px;
+    font-size: 14px;
+    line-height: 16px;
+    background-color: #00acdc;
+    color: #fff;
+    border: 1px solid #00acdc;
+
+    &:hover {
+      color: $color-white;
+      background-color: $color-blue-accent;
```

Comment:
Ensure that the `$color-white` and `$color-blue-accent` variables are defined in your SCSS files. If they're not defined, this could lead to a compilation error.